From c7f958df922ef08266b10d89f5fdaca5fcd4a664 Mon Sep 17 00:00:00 2001
From: Christian Biermann <bie@iesy.com>
Date: Thu, 2 Mar 2023 13:32:23 +0100
Subject: [PATCH 1/1] arm64: dts: rockchip: px30: remove adc-keys backlight
 battery charger

remove all device tree objects which have no corresponding hardware on RPX30 OSM-SF
---
 arch/arm/dts/px30-evb.dts | 109 --------------------------------------
 1 file changed, 109 deletions(-)

diff --git a/arch/arm/dts/px30-evb.dts b/arch/arm/dts/px30-evb.dts
index d955e1a598..66792b7702 100644
--- a/arch/arm/dts/px30-evb.dts
+++ b/arch/arm/dts/px30-evb.dts
@@ -13,84 +13,6 @@
 	model = "Rockchip PX30 EVB";
 	compatible = "rockchip,px30-evb", "rockchip,px30";
 
-	adc-keys {
-		u-boot,dm-pre-reloc;
-		compatible = "adc-keys";
-		io-channels = <&saradc 2>;
-		io-channel-names = "buttons";
-		keyup-threshold-microvolt = <1800000>;
-
-		vol-up-key {
-			u-boot,dm-pre-reloc;
-			linux,code = <KEY_VOLUMEUP>;
-			label = "volume up";
-			press-threshold-microvolt = <10000>;
-		};
-
-		vol-down-key {
-			linux,code = <KEY_VOLUMEDOWN>;
-			label = "volume down";
-			press-threshold-microvolt = <170000>;
-		};
-
-		home-key {
-			linux,code = <KEY_HOME>;
-			label = "home";
-			press-threshold-microvolt = <254000>;
-		};
-
-		menu-key {
-			linux,code = <KEY_MENU>;
-			label = "menu";
-			press-threshold-microvolt = <414000>;
-		};
-
-		esc-key {
-			linux,code = <KEY_ESC>;
-			label = "esc";
-			press-threshold-microvolt = <614000>;
-		};
-	};
-
-	backlight: backlight {
-		compatible = "pwm-backlight";
-		pwms = <&pwm1 0 25000 0>;
-		brightness-levels = <
-			  0   1   2   3   4   5   6   7
-			  8   9  10  11  12  13  14  15
-			 16  17  18  19  20  21  22  23
-			 24  25  26  27  28  29  30  31
-			 32  33  34  35  36  37  38  39
-			 40  41  42  43  44  45  46  47
-			 48  49  50  51  52  53  54  55
-			 56  57  58  59  60  61  62  63
-			 64  65  66  67  68  69  70  71
-			 72  73  74  75  76  77  78  79
-			 80  81  82  83  84  85  86  87
-			 88  89  90  91  92  93  94  95
-			 96  97  98  99 100 101 102 103
-			104 105 106 107 108 109 110 111
-			112 113 114 115 116 117 118 119
-			120 121 122 123 124 125 126 127
-			128 129 130 131 132 133 134 135
-			136 137 138 139 140 141 142 143
-			144 145 146 147 148 149 150 151
-			152 153 154 155 156 157 158 159
-			160 161 162 163 164 165 166 167
-			168 169 170 171 172 173 174 175
-			176 177 178 179 180 181 182 183
-			184 185 186 187 188 189 190 191
-			192 193 194 195 196 197 198 199
-			200 201 202 203 204 205 206 207
-			208 209 210 211 212 213 214 215
-			216 217 218 219 220 221 222 223
-			224 225 226 227 228 229 230 231
-			232 233 234 235 236 237 238 239
-			240 241 242 243 244 245 246 247
-			248 249 250 251 252 253 254 255>;
-		default-brightness-level = <200>;
-	};
-
 	vcc_phy: vcc-phy-regulator {
 		compatible = "regulator-fixed";
 		regulator-name = "vcc_phy";
@@ -359,37 +281,6 @@
 				regulator-name = "vcc3v3_lcd";
 			};
 		};
-
-		battery {
-			compatible = "rk817,battery";
-			ocv_table = <3500 3625 3685 3697 3718 3735 3748
-						3760 3774 3788 3802 3816 3834 3853
-						3877 3908 3946 3975 4018 4071 4106>;
-			design_capacity = <2500>;
-			design_qmax = <2750>;
-			bat_res = <100>;
-			sleep_enter_current = <300>;
-			sleep_exit_current = <300>;
-			sleep_filter_current = <100>;
-			power_off_thresd = <3500>;
-			zero_algorithm_vol = <3850>;
-			max_soc_offset = <60>;
-			monitor_sec = <5>;
-			sample_res = <10>;
-			virtual_power = <1>;
-		};
-
-		charger {
-			compatible = "rk817,charger";
-			min_input_voltage = <4500>;
-			max_input_current = <1500>;
-			max_chrg_current = <1300>;
-			max_chrg_voltage = <4200>;
-			chrg_term_mode = <1>;
-			chrg_finish_cur = <300>;
-			virtual_power = <0>;
-			dc_det_adc = <0>;
-		};
 	};
 };
 
-- 
2.30.2

